src/main/java/frc/team2412/robot/commands/launcher/FullTargetCommand.java

```diff
@@ -81,6 +84,10 @@ public void execute() {
 			intakeSubsystem.feederStop();
 		}
 
+		if (isAuto && launcherSubsystem.isAtSpeed()) {
```

Comment:
Instead of adding a separate parameter, you can just pass `isAtSpeed()` as the boolean supplier to the auto commands.

Reply:
![image](https://github.com/robototes/Crescendo2024/assets/145405364/13c3770a-10a0-4af1-9506-e92c1951994d)
So like this?

Reply:
Resolved, i just deleted all my changes here bcs you told me a workaround